build.cake

```diff
@@ -747,6 +767,15 @@ Task("Publish-NuGet")
     Information("Publish-NuGet Task failed, but continuing with next Task...");
 });
 
+Task("Publish-Coverage")
+    .IsDependentOn("Run-Unit-Tests")
+    .WithCriteria(() => isRunningOnAppVeyor)
+    .Does(() =>
+{
+    CoverallsNet(artifactsDir.GetFilePath(new FilePath("./coverage.xml")), CoverallsNetReportType.OpenCover,
+        new CoverallsNetSettings {RepoToken = "jKaVvndtCVJ2y7pNwQtYeHPn2jAxUpuyh", UseRelativePaths = true, ServiceName = "AppVeyor"});
```

Comment:
This should be set as env variable like: COVERALLS_REPO_TOKEN and probably as secure?